app/templates/buyers/brief_overview.html

```diff
@@ -38,7 +38,28 @@
           </form>
         </div>
       {% endif %}
-   {% endblock %}
+    {% endblock %}
+
+    {% with messages = get_flashed_messages(with_categories=true) %}
+        {% if messages %}
+            {% for category, message in messages %}
+                {% if category == 'error' %}
+                    <div class="banner-destructive-without-action">
```

Comment:
Is there not some sort of macro that means we don't need to write the HTML?

Example: https://github.com/alphagov/digitalmarketplace-brief-responses-frontend/blob/master/app/templates/briefs/view_response_result.html

Reply:
👍